### PR TITLE
rename optIntoAsa to optInToAsa

### PR DESCRIPTION
**Pull request by sczembor**
Closes #x

## Proposed Changes

- Renamed 'optIntoAsa' to 'optInToAsa'

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ Features, Bug Fixes, API Breaking, Deprecated, Infrastructure, Template Updates
 
 ### Bug Fixes
 
+- Renamed `optIntoAsa` to `optInToAsa` to remain naming convention consistency across the project.
```
Comment (vuvoth) [inline CHANGELOG.md:25]: It's should be in ###Breaking changes I think

**Review by vuvoth — APPROVED** [review]
LGTM! Don't forgot update in algob and documents!

File: CHANGELOG.md (hunk 1 above)
Comment (sczembor) [reply CHANGELOG.md:25]: you are right!

Comment (sczembor) [inline CHANGELOG.md:25]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -22,6 +22,8 @@ Features, Bug Fixes, API Breaking, Deprecated, Infrastructure, Template Updates
 
 ### Bug Fixes
 
+- Renamed `optIntoAsa` to `optInToAsa` to remain naming convention consistency across the project. 
+
 ### Breaking Changes
```
Comment (sczembor) [inline CHANGELOG.md:28]:
```suggestion

- Renamed `optIntoAsa` to `optInToAsa` to remain naming convention consistency across the project. 
```

**Pull request merged**